### ISSUE TITLE
Upscaled app icon on Mac - poor quality

### ISSUE THREAD

**Issue by Lukaszeq** [cyanfish/naps2#174]
I know it sounds like a first-world problem, but is there a raw, editable file of the icon? I would be more than happy to make it spic and span on Mac. Thanks for the heads up.

**Comment by cyanfish** [comment]
You are absolutely right, I didn't realize that (as I didn't have magnification on in the dock). The way macOS does scaling seems to be a bit strange as the highest-resolution icon does look better than that, but I was able to fix it by manually adding an upscaled icon. This should be fixed in the next version.
